crates/router/src/connector/cryptopay.rs

```diff
@@ -288,13 +297,21 @@ impl ConnectorIntegration<api::Authorize, types::PaymentsAuthorizeData, types::P
             .change_context(errors::ConnectorError::ResponseDeserializationFailed)?;
         event_builder.map(|i| i.set_response_body(&response));
         router_env::logger::info!(connector_response=?response);
+        let capture_amount_core = match response.data.price_amount {
```

Comment:
can we rename it to `capture_amount_in_minor_units`

Reply:
Done